Pull request for linear scaling parameters and more added in longDesc

kblondal opened the pull request:
Added in longDesc of surfaceThermoPt.py: DFT binding energy, linear scaling parameters, 2D gas model comments and authors. 
Added in longDesc of adsorptionPt.py: DFT binding energy, linear scaling parameters, 2D gas model comments.
Added four species in adsorptionPt.py that were accidentally left out before. These are the entries with indices 84-87. Their labels are also in the tree now.

kblondal committed:
Added longDesc details to surfaceThermoPt library.

Details include authors, DFT binding energy, linear scaling parameters and 2D gas model comments.

kblondal committed:
Added longDesc details in adsorptionPt thermo groups database.

Details include DFT binding energy, linear scaling parameters and 2D gas model comments.

kblondal committed:
Added four species to adsorptionPt thermo groups database.

Added four species that were accidentally left out (entries with indices 84-87).
Added their labels in the tree.

rwest reviewed (commented):
Is the first commit intended? what is this `.surfaceThermoPt.py.swp` file?

rwest commented:
Perhaps ideally the new libraries should be a different pull request.

kblondal commented:
I thought I should add the file in the first commit because it showed up as an "untracked file" when I checked git status, and I wasn't sure what it meant. It should probably not be included.
I have reverted the first commit and also the ones adding the new reaction libraries. No problem, I will make a separate pull request for them. Thanks for letting me know.

kblondal committed:
Added a Pt reaction library based on Deutschmann 2006 paper

Fixed error in paper for reaction 16

kblondal committed:
Added a Pt reaction library based on Vlachos(2007) paper

Fixed a species name typo in two places

rwest committed:
Try removing the eley-rideeal reaction with StickingCoefficient kinetics.

Something is causing the ODE solver to crash, probably with super-stiff
kinetics. Perhaps because we're calculating this rate wrong. Just a hunch...

rwest committed:
Unit changes and corrections to Deutschmnann2006 reaction library.

Changing units to match was in the paper.
 (they were in cm, not m)
Fixed units for a termolecular (bidentate) dissociative adsorption
 (should be cm^5/mol^2/s)
Commented out 32 instead of 31
 (they are reverses of each other; we want the adsorption)

rwest committed:
Revert "Try removing the eley-rideeal reaction with StickingCoefficient kinetics."

This reverts commit 0d53e767c7bd0b00fe027923fd87ac600e737bd7.

kblondal committed:
Uncommented methane dissociative adsorption (R3)

And commented out reaction 34, which is the reverse of reacion 3.

kblondal committed:
Merge branch 'cat' of https://github.com/rwest/RMG-database into cat

kblondal committed:
Commented out R-* in adsorptionPt

It is just H with a single bond to Pt, which is already in the thermo database.

kblondal committed:
Fixed two errors in Vlachos2007 dictionary

Changed HCOO from charged to radical and changed OCHXOX to OXCHOX to be consistent with the reaction library.

kblondal committed:
Merge branch 'cat' of https://github.com/rwest/RMG-database into cat

kblondal committed:
Merge branch 'cat' of https://github.com/rwest/RMG-database into cat

kblondal committed:
Corrected NN_ads bond type to Pt surface

Changed to vdW bond in adjacency lists of surfacethermoPt and adsorptionPt file.
Fixed a slight difference that makes in the thermo and adsorption corrections for adsorbed NN.
Fixed linear scaling comments.
Commented out NN entry in adsorptionPt file, since it is not needed, and removed N-*N-* node in tree.

kblondal committed:
Added thermo for CH2O-h

Added entries for for CH2O-h in surfacethermoPt and adsorptionPt files (includes adding C-*R2O-* node in tree).

kblondal committed:
Removed merge conflict messages from Deutschmann reaction library.

kblondal committed:
Fixed syntax errors

Fixed syntax error in the NN new adjacency list and in the new H2CO-h thermo entry in surfacethermoPt.
Resolved merge conflicts in adsorptionPt.

kblondal committed:
Commented out incorrect entry in surfacethermoPt.

More specifically, index 62 was commented out since it should be analogous to index 51.

rwest commented:
I rebased the last few commits onto `cat` (and `catmerge`) and split the syntax errors commit into fixup commits which merged with the original commits. I think the only difference between your `cat` and my `cat` is now a few whitespace changes. If you could do a hard reset and force push to get them back on track, it'll simplify future merges and pull requests.

It would be something like:
```
git fetch richard
git checkout -B cat richard/cat
git push -f github cat
```
(assuming your remotes are called `richard` for mine and `github` for yours).

kblondal commented:
Done, thank you.
